### ISSUE TITLE
Build issue on version 4.3.3 - duplicate entry: fm/icelink/audioprocessing/Native.class

### ISSUE THREAD

**Issue by tfcporciuncula** [idnow/de.idnow.android#68]
My project builds fine with 4.2.2, but it fails with 4.3.3 with the following message:

```
Execution failed for task ':app:checkDebugDuplicateClasses'.
> Could not resolve all files for configuration ':app:debugRuntimeClasspath'.
   > Failed to transform idnow-android-sdk-4.3.3.aar (de.idnow.sdk:idnow-android-sdk:4.3.3) to match attributes {artifactType=enumerated-runtime-classes, org.gradle.category=library, org.gradle.libraryelements=jar, org.gradle.status=release, org.gradle.usage=java-runtime}.
      > Execution failed for AarToClassTransform: /Users/fred/.gradle/caches/transforms-2/files-2.1/aa7a82b7cf8cc3273da565e163ee74c0/jetified-idnow-android-sdk-4.3.3.aar.
         > duplicate entry: fm/icelink/audioprocessing/Native.class

Cause: duplicate entry: fm/icelink/audioprocessing/Native.class
```

If you create a new project and add the IDnow dependency, it'll fail with this error, so it's really easy to reproduce.

**Comment by kondratk** [comment]
Got the same issue.

**Comment by tfcporciuncula** [comment]
It's actually been resolved for me on 4.3.4, I was about to close this issue. Does it still happen for you on the latest version @kondratk?

**Comment by kondratk** [comment]
I'll check it and let you know. Thanks for the info!

**Comment by kondratk** [comment]
@tfcporciuncula works for me!

**Comment by tfcporciuncula** [comment]
Seems to have been resolved on 4.3.4.
